OsmAnd/src/net/osmand/plus/activities/MapActivityActions.java

```diff
@@ -142,19 +145,24 @@ public MapActivityActions(MapActivity mapActivity) {
 		settings = mapActivity.getMyApplication().getSettings();
 		routingHelper = mapActivity.getMyApplication().getRoutingHelper();
 		drawerLogoHeader = new ImageView(mapActivity);
-		drawerLogoHeader.setPadding(-AndroidUtils.dpToPx(mapActivity, 8f), AndroidUtils.dpToPx(mapActivity, 16f), 0,0);
-		drawerOsmAndFooter = mapActivity.getLayoutInflater().inflate(R.layout.powered_by_osmand_item, null);
+		drawerLogoHeader
+				.setPadding(-AndroidUtils.dpToPx(mapActivity, 8f), AndroidUtils.dpToPx(mapActivity, 16f), 0,
+						0);
+		drawerOsmAndFooter = mapActivity.getLayoutInflater()
+				.inflate(R.layout.powered_by_osmand_item, null);
 	}
 
 	public void addAsTarget(double latitude, double longitude, PointDescription pd) {
 		TargetPointsHelper targets = getMyApplication().getTargetPointsHelper();
-		targets.navigateToPoint(new LatLon(latitude, longitude), true, targets.getIntermediatePoints().size() + 1,
+		targets.navigateToPoint(new LatLon(latitude, longitude), true,
```

Comment:
Why these many extra line feeds added? You may do that in new code, but do not touch existing.